celery/contrib/rdb.py

```diff
@@ -124,6 +124,7 @@ def get_avail_port(self, host, port, search_limit=100, skew=+0):
         this_port = None
         for i in range(search_limit):
             _sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            _sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
Actually, according to MSDN, it's available since Windows 95. So there's no problem there. https://msdn.microsoft.com/en-us/library/windows/desktop/ms740621(v=vs.85).aspx